types/ffmpeg__libav-core/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "esModuleInterop": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
```

Comment:
you would also have to add paths translation here for scoped module to be properly resolved from outside:
https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages

Reply:
Added path to tsconfig.json, thanks!